### DISCUSSION TITLE
Delivroo Bug issues

### DISCUSSION THREAD

**Discussion by OmarChefrad** [OmarChefrad/Delivroo#4] [Q&A] [answered]
i dont know how to manage the image assets for diffrent platform such in ios pics are normal but in android they start bugging

**Comment by Astronier** [comment, marked as answer]
you have to look for a solution which we found in the documentation: it was needed to distribute the images as [static resources](https://reactnative.dev/docs/images#static-image-resources) and use the native way of handling images. That means choosing the right one by the screen density.

**Reply by OmarChefrad** [reply]
thanks for the help
